app/controllers/application_controller.rb

```diff
@@ -1,2 +1,16 @@
 class ApplicationController < ActionController::API
+
+  def current_user
```

Comment:
yeah ruby is snake_case for methods and vars, CamelCase for classes and modules.